flare/env.py

```diff
@@ -32,40 +76,84 @@ def __init__(self, structure: Structure, atom: int, cutoffs, sweep=1):
         self.ctype = structure.coded_species[atom]
 
         self.cutoffs = np.copy(cutoffs)
+        self.cutoffs_mask = cutoffs_mask
+
+        self.setup_mask()
+
+        assert self.scalar_cutoff_3 <= self.scalar_cutoff_2, \
+            "2b cutoff has to be larger than 3b cutoff"
+        # # TO DO, once the mb function is updated to use the bond_array_2
+        # # this block should be activated.
+        # assert self.scalar_cutoff_mb <= self.scalar_cutoff_2, \
+        #         "mb cutoff has to be larger than mb cutoff"
 
         self.compute_env()
 
+    def setup_mask(self):
+
+        self.scalar_cutoff_2, self.scalar_cutoff_3, self.scalar_cutoff_mb, \
+            self.cutoff_2b, self.cutoff_3b, self.cutoff_mb, \
+            self.nspecie, self.n2b, self.n3b, self.nmb, self.specie_mask, \
```

Comment:
On a separate note, can we more verbosely name n2b and n3b?

Reply:
n2b_groups? n3b_groups?
n2body_groups? n3body_groups?

Reply:
omg, that's a long name. Sure. we can. will do it in the refactor branch...

Reply:
`n2b_groups` is my vote ;)